### PR TITLE
Update DC Universe Online, removed Linux from testing operating systems

### PR DESCRIPTION
**Pull request by Zemogiter**
Game was run for 4 hours straight and with that I assume the crashes I was getting last time are solved.

### CODE REVIEW

File: Applications/Games/DC Universe Online/Online/script.json [hunk 1]
```diff
@@ -4,9 +4,7 @@
     "compatibleOperatingSystems" : [
         "LINUX"
     ],
-    "testingOperatingSystems" : [
-        "LINUX"
-    ],
-    "free"          : true,
-    "requiresPatch" : false
+    "testingOperatingSystems" : [],
```
Comment (ImperatorS79) [inline Applications/Games/DC Universe Online/Online/script.json]: Could you add macOS in testing? I do not see anything that would prevent the game from running under macos.

Comment (Zemogiter) [reply Applications/Games/DC Universe Online/Online/script.json]: Why not in stable?

Comment (ImperatorS79) [reply Applications/Games/DC Universe Online/Online/script.json]: Do you have a mac to be sure it works ?

Comment (Zemogiter) [reply Applications/Games/DC Universe Online/Online/script.json]: No.

Comment (ImperatorS79) [reply Applications/Games/DC Universe Online/Online/script.json]: So testing ^^.

**Pull request merged**